README.md

```diff
@@ -176,6 +176,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 
 - ### **A**
 
+  - [Ahmad Hassan](https://github.com/ahmadhassan7)
```

Comment:
Please add your name in line 245, after Ahamed Basha N.
Let us know if you need any help with that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
I did so from mobile though hope it's correct 😅